components/script/dom/bindings/reflector.rs

```diff
@@ -74,6 +74,7 @@ impl Reflector {
 }
 
 /// A trait to provide access to the `Reflector` for a DOM object.
+#[must_root]
```

Comment:
Why is this necessary?

Reply:
Like mentioned before - it is just some heuristics of mine.